shared/actions/devices.js

```diff
@@ -103,11 +112,20 @@ export function removeDevice (deviceID: string, name: string, currentDevice: boo
       revokeRevokeDeviceRpc({
         param: {deviceID, force: false},
         callback: error => {
-          dispatch({
-            type: Constants.deviceRemoved,
-            payload: error,
-            error: !!error,
-          })
+          if (error) {
+            dispatch({
+              type: Constants.deviceRemoved,
+              payload: error,
+              error: true,
+            })
+          } else {
+            dispatch({
+              type: Constants.deviceRemoved,
+              payload: error,
+              error: false,
```

Comment:
payload is usually not null: https://github.com/keybase/client/blob/master/shared/constants/types/flux.js#L6

When are you getting into these cases?